python/protoc-gen-mypy

```diff
@@ -40,6 +40,42 @@ else:
 GENERATED = "@ge" + "nerated"  # So phabricator doesn't think this file is generated
 HEADER = "# {} by generate_proto_mypy_stubs.py.  Do not edit!\n".format(GENERATED)
 
+# See https://github.com/dropbox/mypy-protobuf/issues/73 for details
+PYTHON_RESERVED = set([
```

Comment:
Might want to add `async` and `await` for Python 3.

Reply:
Can `keyword.iskeyword` be used for this?